docs-guides/source/opt-quantization-api.md

```diff
@@ -162,21 +162,21 @@ model.eval()
 quantized_model = quantizer.finalize()
 ```
 
-Note that, here, we set the first and last values of the ``milestones`` parameter to ``0``. 
-The first milestone turns on observes, and setting it to zero ensures that we start measuring 
-quantization statistics from step 0. And the last milestone applies batch norm in inference mode,
+Note that here we set the first and last values of the ``milestones`` parameter to ``0``. 
+The first milestone turns on observers, and setting it to zero ensures that we start measuring 
+quantization statistics from step 0. The last milestone applies batch norm in inference mode,
 which means we do not use the calibration data to update the batch norm statistics. We do this because
 we do not want training data to influence the batch norm values. The other two milestones 
 are used to control when fake quantization simulation is turned on and when observers are turned off.
-We can set them to values larger than 0 so that they are never turned on.
+We can set them to values larger than zero so that they are never turned on.
 
 
 #### Quantization Aware Training (QAT)
 
 We use [``LinearQuantizer``](https://apple.github.io/coremltools/source/coremltools.optimize.torch.quantization.html#coremltools.optimize.torch.quantization.LinearQuantizer)
 here as well, with a few extra steps, as demonstrated below. 
 
-Specify config in a ``YAML`` file:
+Specify config in a ``yaml`` file:
```

Comment:
It's an acronym. So I think we want it all in caps. I think that's the only way I've seen it.

However, in that case maybe we don't want the backticks. The file extension is lowercase. So I think it should probably either be "YAML", or backticks and ".yaml".

Reply:
You're right, the [Apple Style Guide](https://help.apple.com/pdf/applestyleguide/en_US/apple-style-guide.pdf) says "Don’t use filename extensions to refer to file types." So I will change to YAML with no backticks.